### ISSUE TITLE
Is it possible to provide a valid Scaleway / Backblaze .env config

### ISSUE THREAD

**Issue by fung1** [QingWei-Li/notea#138]
I need help with configurating Scaleway / Backblaze. Thanks for the app!

**Comment by CDFN** [comment]
Backblaze's B2 is S3 compatible. Just provide credentials they give you.

**Comment by chenryace** [comment]
> 

No, B2 works, but it is easy to crash, expecially when there is a hint/warn in one of your note. you need to write some rule for it. I failed.
